File: Lesson/src/lesson7/enums/SubMenuBuilder.java
package lesson7.enums;

import MenuBuilder.ConsoleMenuBuilder;
import lesson7.enums.actions.CurrenciesEnumAction;
import lesson7.enums.actions.FlowersEnumAction;

public class SubMenuBuilder {
    static public ConsoleMenuBuilder getMenu(String prnTitle) {
        final String FLOWERS = "Flowers";
        final String CURRENCIES = "Currencies";


        return new ConsoleMenuBuilder(prnTitle)
                .AddMenuItem("1", FLOWERS, new FlowersEnumAction())
                .AddMenuItem("2", CURRENCIES, new CurrenciesEnumAction())
                .AddMenuItem("0", "Go to previous menu", null, true);
    }
}

File: Lesson/src/lesson13/SubMenuBuilder.kt
package lesson13

import MenuBuilder.ConsoleMenuBuilder
import lesson13.actions.Lesson13

class SubMenuBuilder {
    companion object {
        @JvmStatic
        fun getMenu(prnTitle: String): ConsoleMenuBuilder {
            val LESSON13 = "Lesson13"

            return ConsoleMenuBuilder(prnTitle)
                    .AddMenuItem("1", LESSON13, Lesson13())
                    .AddMenuItem("0", "Go to previous menu", null, true)
        }
    }
}